src/main/java/io/fabric8/jenkins/openshiftsync/JobProcessor.java

```diff
@@ -1,7 +1,7 @@
 package io.fabric8.jenkins.openshiftsync;
 
 
-import static io.fabric8.jenkins.openshiftsync.Annotations.DISABLE_SYNC_CREATE;
+import static io.fabric8.jenkins.openshiftsync.Annotations.*;
```

Comment:
By convention, we do not use wildcard imports or at least we are trying to remove them.